source/Public/Workspace/Update-FabricWorkspaceRoleAssignment.ps1

```diff
@@ -51,9 +51,7 @@ function Update-FabricWorkspaceRoleAssignment
     try
     {
         # Step 1: Ensure token validity
-        Write-Message -Message "Validating token..." -Level Debug
         Test-TokenExpired
-        Write-Message -Message "Token validation completed." -Level Debug
 
         # Step 2: Construct the API URL
         $apiEndpointUrl = "{0}/workspaces/{1}/roleAssignments/{2}" -f $FabricConfig.BaseUrl, $WorkspaceId, $WorkspaceRoleAssignmentId
```

Comment:
does this need the base url?

Reply:
There are 140 places with this approach. Are you sure you want me to change all of them now? 
I was aware of it as new function (`Invoke-FabricRestMethod`) can optionally add baseURL.
However, at this point, I'd accept baseUrl to exist for some calls (using global `$FabricConfig.baseUrl` would be mandatory though) in transition period. For new functions we should avoid providing it.
Further more, let's complete this PR (blocker for further changes), and we can change this under one of these issues:
#87 or #85 
Deal? :)

Reply:
Should it be a public function? are there use cases where people will want to use this instead of the FabricTools commands?

Reply:
Yes. For example where the new API appear, but function in FabricTools has not yet been implemented. 
I personally used it multiple times when working on new features or testing API with various configurations.